draft-ietf-taps-interface.md

```diff
@@ -882,8 +882,8 @@ being interpreted as a higher priority anywhere.]
 Type:
 : Boolean
 
-If true, it specifies that a Message should be delivered to the other side after the previous Message which was passed to the same Connection via the Send
-Action. If false, the Message may be delivered out of order.
+If true, it specifies that the receiver-side transport system should only deliver the Message to the receiving application after the previous ordered Message which was passed to the same Connection via the Send
```

Comment:
From my perspective, this is a minor knit. Perhaps try the following instead:

> If true, it specifies that the receiver-side transport system is expected to only deliver the Message to
> the receiving application after the previous ordered Message which was passed to the same Connection.
> The actual signaling of this is protocol specific.

Reply:
What you say is true, but then TAPS uses transports that also have receiver sematics: 

True - seems like receiver TCP semantics? (so if there is a TCP receiver at the remote, TAPS can use that). SCTP can also support this.

False - seems like UDP. If the stack supports this, the SCTP can alternatively do this.

... My choice would be I'd prefer to standardise "does not need to be ordered", if I had a choice of semantics.

Reply:
Indeed, the sender side alone can never guarantee ordering - at best, it can know that there is a receiver that takes care of it, or (e.g. in case of SCTP) signal the wish to the receiver. I updated the text to talk about the receiver behavior to avoid indicating sender-side logic that shouldn't be there. I thought the receiver-oriented text is clearer.

Reply:
I agree with what you're saying, it's clearer.
My only comment (perhaps a nit) is to change "receiver-side transport system" to "receiver-side transport protocol stack", so we do not imply that there is a transport system (which, I think, we define as a TAPS implementation) running there, it could also just be a vanilla stack.

Reply:
I agree, that's better